site/src/components/BuildsTable/BuildsTable.tsx

```diff
@@ -79,7 +79,9 @@ export const BuildsTable: FC<BuildsTableProps> = ({ builds, className }) => {
                   </span>
                 </TableCellLink>
                 <TableCellLink to={buildPageLink}>
-                  <span style={{ color: status.color }}>{status.status}</span>
+                  <span style={{ color: status.color }} className={styles.status}>
```

Comment:
Now that this has a class, we can move the inline style into that class.

Reply:
It is inline because it is dynamic and passed from the props.

Reply:
we can still pass it i think, but nbd if you want to leave it inline!